orchestrator.Jenkinsfile

```diff
@@ -59,6 +59,7 @@ projects = [
     "tests/gateway-tests": "gateway-acceptance-tests",
     "tests/test-environments": "test-environments",
     "tests/kubeless-test-client": "kubeless-test-client",
+    "tests/kubeless": "kubeless-tests",
```

Comment:
Why the folder name is different than the component name, we should target a consistent approach